recipes/fuse-overlayfs/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: fuse-overlayfs
+  version: {{ version }}
+
+source:
+  - url: https://github.com/containers/fuse-overlayfs/archive/v{{ version }}.tar.gz
+    sha256: 5df218732244059057686194b0e1fef66fb822d4087db48af88e1bc29bb1afde
+    patches:
+      - 0001-Add-missing-defines-for-CentOS-6.patch
+      - 0002-Fix-version-tag.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf >=2.69
+    - automake >=1.9
+    - make
+    - pkg-config >=0.9.0
+  host:
+    - libfuse >=3.2.1
```

It's probably better not to put a version constraint here as it will override any global pins on the version and cause the latest `libfuse` to always be used:
```suggestion
    - libfuse
```

---

Right, that was careless -- I should know better not to do these mistakes :).
I've left the minimum version constraint in place and added an additional one for `conda-build` to apply a possible future pinning to.